stdlib/operator.pyi

```diff
@@ -177,3 +181,7 @@ def itruediv(__a: Any, __b: Any) -> Any: ...
 def __itruediv__(a: Any, b: Any) -> Any: ...
 def ixor(__a: Any, __b: Any) -> Any: ...
 def __ixor__(a: Any, b: Any) -> Any: ...
+
+if sys.version_info >= (3, 11):
+    def call(__obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
+    def __call__(obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
```

Comment:
mypy doesn't support `ParamSpec` yet:

```suggestion
    def call(__obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...  # type: ignore
    def __call__(obj: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...  # type: ignore
```

Reply:
Any idea why it doesn't seem to be causing any errors in the CI tests? I was surprised it seemed to work as well as it did.

Reply:
The CI doesn't run Python 3.11 yet, so everything under `if sys.version_info >= (3, 11):` is untested for now.

Reply:
Hah, makes sense.